assets/js/googlesitekit/data/create-settings-store.test.js

```diff
@@ -96,10 +95,9 @@ describe( 'createSettingsStore store', () => {
 
 		describe( 'fetchGetSettings', () => {
 			it( 'does not require any params', () => {
-				muteFetch( /^\/google-site-kit\/v1\/core\/site\/data\/settings/ );
```

Comment:
Please keep all of these and update any new fetch mocks that were added solely for the purpose of preventing errors.